ha/resource/dynamic_fid_service_ra.py

```diff
@@ -92,7 +92,10 @@ def _get_systemd_service(self) -> str:
         fid_service_name: str = res_param["OCF_RESKEY_fid_service_name"]
         local_node: str = res_param["OCF_RESKEY_CRM_meta_on_node"]
         resource: str = res_param["OCF_RESOURCE_INSTANCE"]
-        instance_id: int = int(resource.split('-')[-1])
+        resource_instance: str = str(resource.split('-')[-1])
+        instance_id: int = 1
```

Comment:
Why default should be 1? In which case, it's not present in res_param?

Reply:
i have considered resource name as `mort-confd-1` `s3server-1`  ....
where last `1` considered as instance id
but setup failing as resource name was `motr-confd` where we not have instance id so default treated as 1

Reply:
Added resource definition and redefine resource name.